model-optimizer/mo/front/extractor.py

```diff
@@ -131,6 +131,16 @@ def attr_getter(node: Node, name: str):
     return None
 
 
+def get_boolean_attr(node: Node, attr: str):
+    if node[attr] in ['true', 'false']:
```

Comment:
Why we compare with string values `'true'` and `'false'`? It is not valid value.

Reply:
Yes, it was a part of deleted scenario, now this function updated.